persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/Conversion.java

```diff
@@ -148,6 +148,9 @@
                   && ct != Column.Type.List
```

Comment:
Could these big checks be simplified with `!ct.isComplexType()`? or something similiar.

Reply:
sure. will do in a follow-up PR.